ompi/mca/pml/ob1/pml_ob1_isend.c

```diff
@@ -268,6 +269,16 @@ int mca_pml_ob1_send(const void *buf,
         rc = sendreq->req_send.req_base.req_ompi.req_status.MPI_ERROR;
     }
 
+    if (sendreq->req_send.req_base.req_pml_complete) {
```

Comment:
Why here and not in mca_pml_ob1_send_request_fini, a function called by all request completions ?